### ISSUE TITLE
can not load simple_model.earth

### ISSUE THREAD

**Issue by x1244** [gwaldron/osgearth#1872]
When i try load simple_model.earth, I get the message
```
PS D:\osgearth\tests> osgearth_viewer .\simple_model.earth
void StateSet::setGlobalDefaults() ShaderPipeline disabled.
[osgEarth]* [GeoPoint] ILLEGAL: called GeoPoint::toWorld with AltitudeMode = RELATIVE_TO_TERRAIN
```
How can I do for this?

osgEarth version
```
PS D:\osgearth\tests> osgearth_version --caps
[osgEarth]  [Registry] Note: GDAL_DATA environment variable is not set
[osgEarth]  [Capabilities] Capabilities:
[osgEarth]  [Capabilities]   osgEarth Version:  3.2.0 build 119
[osgEarth]  [Capabilities]   OSG Version:       3.7.0
[osgEarth]  [Capabilities]   GDAL Version:      3.3.0dev
[osgEarth]  [Capabilities]   GEOS Version:      3.10.0dev
[osgEarth]  [Capabilities]   GPU Vendor:        NVIDIA Corporation
[osgEarth]  [Capabilities]   GPU Renderer:      GeForce RTX 2070 SUPER/PCIe/SSE2
[osgEarth]  [Capabilities]   GL/Driver Version: 4.6.0 NVIDIA 456.71
[osgEarth]  [Capabilities]   GL Core Profile:   no
```

**Comment by gwaldron** [comment]
Earth files works. But, the error message comes from GeoPoint::toWorld() trying to set a center for paging without access to the Terrain. Workarounds include
* ignore the message
* set `paged="false"`
* use a Model annotation instead

**Comment by x1244** [comment]
It was fixed at version 3.3.0 build147.
```
> osgearth_version --caps
[osgEarth]  [Registry] Note: GDAL_DATA environment variable is not set
[osgEarth]  [Capabilities] Capabilities:
[osgEarth]  [Capabilities]   osgEarth Version:  3.3.0 build 147
[osgEarth]  [Capabilities]   OSG Version:       3.7.0
[osgEarth]  [Capabilities]   GDAL Version:      3.7.0dev-7a4abb688f
[osgEarth]  [Capabilities]   GEOS Version:      3.12.0dev
[osgEarth]  [Capabilities]   GPU Vendor:        NVIDIA Corporation
[osgEarth]  [Capabilities]   GPU Renderer:      NVIDIA GeForce RTX 2070 SUPER/PCIe/SSE2
[osgEarth]  [Capabilities]   GL/Driver Version: 3.3.0 NVIDIA 527.56 (330)
[osgEarth]  [Capabilities]   GL Core Profile:   yes
```
